assets/data-port/import/data/actions.js

```diff
@@ -175,22 +183,28 @@ export const errorStartImport = ( error ) => ( {
 /**
  * Upload a file for a level.
  *
+ * @param {string}   jobId                 The job identifier.
  * @param {string}   level                 Level identifier.
  * @param {Object}   uploadData            Data to submit.
  * @param {Object}   [options]
- * @param {Function} [options.onSuccess] Step name.
- * @param {Function} [options.onError]   Data to submit.
+ * @param {Function} [options.onSuccess]   Step name.
+ * @param {Function} [options.onError]     Data to submit.
  */
 export function* uploadFileForLevel(
+	jobId,
 	level,
 	uploadData,
 	{ onSuccess, onError } = {}
 ) {
 	yield startFileUploadAction( level, uploadData );
 
 	try {
+		if ( ! jobId ) {
+			jobId = API_SPECIAL_ACTIVE_JOB_ID;
```

Comment:
We could add `jobId: API_SPECIAL_ACTIVE_JOB_ID` to the default state so that it doesn't need to be handled at the action level.

Reply:
Yeah, I had that initially, but some endpoints don't accept `active` (start), so I wasn't sure if we should be explicit.

Reply:
Makes sense, though sounds like that would still need a check for empty `jobId`. 
Also a thing to think about is a reset action that'll be needed on the Done screen for starting a new import. But that would probably work with both approaches.

Reply:
Reset doesn't need any request, just a state reset.

I added a silent early bail on start Import if jobId isn't set d41359192028c33e01f0c6132f7f6c3189c4e38e